### PR TITLE
Migrate Deployment Env

### LINKED ISSUE #107: 【後端】轉移部屬環境

### PR DESCRIPTION
**Pull request by chiashengchen**
resolves #107

### CODE REVIEW

File: .github/workflows/backend_pr_cd_job.yml [hunk 1]
```diff
@@ -44,5 +44,5 @@ jobs:
           key: ${{ secrets.PRIVATE_KEY }}
           port: ${{ secrets.PORT }}
           script: |
-            cd /webApp/backend 
-            sudo docker-compose up -d --build
+            cd ~/citadels/backend 
+            sudo MONGO_URI=$MONGO_URI docker-compose up -d --build
```
Comment (snowmancc) [inline .github/workflows/backend_pr_cd_job.yml:48]: docker-compose.yml 也有設定 URI 了，是因為吃不到所以才在這邊加上的嗎

**Review by snowmancc — APPROVED** [review]
LGTM

File: .github/workflows/backend_pr_cd_job.yml (hunk 1 above)
Comment (chiashengchen) [reply .github/workflows/backend_pr_cd_job.yml:48]: Yes~

**Review by snowmancc — COMMENTED** [review]
LGTM

**Review by timt1028 — APPROVED** [review]
LGTM

**Pull request merged**